src/target/riscv/riscv-013.c

```diff
@@ -3478,14 +3495,13 @@ bool is_mem_access_failed(mem_access_result_t status)
 bool is_mem_access_skipped(mem_access_result_t status)
 {
 	#define MEM_ACCESS_RESULT_HANDLER(name, kind, msg) \
-		case name: return kind == MEM_ACCESS_RESULT_TYPE_SKIPPED;
+		case name: return MEM_ACCESS_RESULT_TYPE_##kind \
+					== MEM_ACCESS_RESULT_TYPE_FAILED;
```

Comment:
This looks like a copy-paste typo (`FAILED -> SKIPPED`): 

```suggestion
		case name: return MEM_ACCESS_RESULT_TYPE_##kind \
					== MEM_ACCESS_RESULT_TYPE_SKIPPED;
```

Reply:
Oh, my bad. Addressed, thank you!